File: 30-cortex-training/python-sdk/cortex-training.py
import os, requests
API = "https://api.costkatana.com/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def upload_training_data(prompts):
    res = requests.post(f"{API}/cortex/training",
        json={"prompts": prompts},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Training data uploaded: {res.json()['data']['count']}")
    return res.json()["data"]

File: 30-cortex-training/npm-package/cortex-training.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function uploadTrainingData(prompts: any[]) {
  const res = await axios.post(`${API}/cortex/training`, {prompts},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Training data uploaded:', res.data.data.count);
  return res.data.data;
}
